php_mcrypt.h

```diff
@@ -2,7 +2,7 @@
    +----------------------------------------------------------------------+
    | PHP Version 7                                                        |
    +----------------------------------------------------------------------+
-   | Copyright (c) 1997-2017 The PHP Group                                |
+   | Copyright (c) 1997-2018 The PHP Group                                |
```

Comment:
This file hasn't changed, so please don't update the year.

Reply:
Reverted php_mcrypt.h